portfolio/src/main/webapp/index.html

```diff
@@ -2,17 +2,116 @@
 <html>
   <head>
     <meta charset="UTF-8">
-    <title>My Portfolio</title>
+    <title>get to know Dez</title>
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
   </head>
   <body>
     <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+        <img src="/desmondsimages/headerpic.png" 
+             alt="Dez Image #1" 
+             style="float:left;"
+              class="image1">
+        <h1>
+             get to know Dez
+        </h1>
+            <p class="leftpad1">
+                <strong>NAME</strong>: Desmond Talton <br><br>
+                <strong>COLLEGE</strong>: The Georgia Institute of Technology <br><br>
+                <strong>MAJOR</strong>: Computer Science <br><br>
+            </p>
+            <button onclick="funFactGenerator()">Random Fun Fact</button>
+            <div id="factgetter">
+                Press to Generate a Fun Fact.
+            </div>    
+        <h>
```

Comment:
`<h>` doesn't seem to be a valid tag, can we remove that? We can do something like:
<img>
`<p><strong>"Hay, my name ih Dehmun Taltun"</strong></p>`

Reply:
Done. Replaced the h tag w/ p tags.

Reply:
As I type this I recognize a mistake I made. Wait. I am about to make another commit. Do not review this code just yet. Its a one line change.

Reply:
Okay its resolved now